pkg/operator/ceph/object/spec.go

```diff
@@ -117,7 +117,9 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) v1.PodTemplateSpec
 	if c.clusterSpec.Network.IsHost() {
 		podTemplateSpec.Spec.DNSPolicy = v1.DNSClusterFirstWithHostNet
 	} else if c.clusterSpec.Network.IsMultus() {
-		k8sutil.ApplyMultus(c.clusterSpec.Network.NetworkSpec, &podTemplateSpec.ObjectMeta)
+		if err := k8sutil.ApplyMultus(c.clusterSpec.Network.NetworkSpec, &podTemplateSpec.ObjectMeta); err != nil {
+			logger.Errorf("failed to apply multus. %v", err)
```

Comment:
return the error

Reply:
> return the error

@travisn I'm not sure if we can return err in  [`deployment.spec.Template`](https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/spec.go#L52) maybe we can define `podTemplateSpec` inside `createdeployment` only as we have for mgr [here](https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/mgr/spec.go#L48l).